vllm/worker/multi_step_model_runner.py

```diff
@@ -316,11 +375,10 @@ def execute_model(
 
         # Pythonize the output and block if needed since it is the last step
         if model_input.is_last_step:
-            outputs = []
-            for output in model_input.cached_outputs:
-                output.pythonize(model_input, self._copy_stream,
-                                 self.pinned_sampled_token_ids)
-                outputs.append(output.sampler_output)
+            outputs = self._final_process_outputs(
+                model_input,
+                model_input.frozen_model_input.async_callback.  # type: ignore
+                keywords["output_proc_callback"])  # type: ignore
```

Comment:
I got `AttributeError: 'NoneType' object has no attribute 'keywords'` error when enabling multi step and disabling the async output processing.
```suggestion
                model_input.frozen_model_input.async_callback.  # type: ignore
                keywords["output_proc_callback"] if model_input.frozen_model_input.async_callback is not None else None)  # type: ignore
```

Reply:
@WoosukKwon thanks for catching this, added